File: Linear Search.c
#include<stdio.h>
int main()
{
    int n,arra[100],count=0,i,search;

    printf("Enter elements in Array: ");
    scanf("%d",&n);

    for(i=0;i<n;i++)
    {
        scanf("%d",&arra[i]);
    }

    printf("Enter the number you want to search");
    scanf("%d",&search);

    for(i=0;i<n;i++)
    {
        if(arra[i]==search)
        {
            printf("%d is present at location %d.\n", search, i+1);
            count++;
        }

    }

    if(count==0)
    {
        printf("Error");
    }

    else
    {
        printf("%d is present %d times in the array.\n",search,count);
    }
    
}

File: Python/Linear Search.py
n=int(input("Enter number of elements:"))

count=0

list=[]

for i in range(0,n):
    elements=int(input())

    list.append(elements)


search=int(input("Enter the number you want to search:"))

for i in range(0,n):
    if list[i]==search:
        print("number is present at index:",i+1)
        count=count+1

if (count==0):
    print("error")
else:
    print(f"{search} is present {count} times in the array")